rust/extractor/src/rust_analyzer.rs

```diff
@@ -128,6 +132,91 @@ impl<'a> RustAnalyzer<'a> {
     }
 }
 
+#[derive(Deserialize)]
+struct CargoManifestMembersSlice {
+    #[serde(default)]
+    members: Vec<String>,
+}
+
+#[derive(Deserialize)]
+struct CargoManifestSlice {
+    workspace: Option<CargoManifestMembersSlice>,
+}
+
+struct ToMlReader {
```

Comment:
Why `ToMl` and not `Toml` ?

Reply:
ah, typo because of a brain fart :laughing: